test/server-custom-livereload.js

```diff
@@ -0,0 +1,120 @@
+import request from 'supertest';
+import assert from 'assert';
+import listen from './helpers/listen';
+import {PassThrough} from 'stream';
+
+describe('tiny-lr', () => {
```

Comment:
Can you change the description to something like `tiny-lr custom livereload` to differentiate it from the other tests in the test output ?